### PR TITLE
Multinode deployment

### PR DESCRIPTION
**Pull request by srisub98**
## Release:

- [ ] To create a release, I will create a PR from to `main` with an incremented version in Chart.yaml
    - Internally, we rely on [chart-releaser](https://github.com/helm/chart-releaser-action) to automatically update the index.yaml on the `gh-pages` branch on version updates.
- [ ] I will **squash and merge** this branch to `main`

### CODE REVIEW

File: .github/workflows/lint-test.yaml
Comment (AndrewHess) [inline .github/workflows/lint-test.yaml:1]: Why are some of these tests removed? Should the `Run chart-testing (list-changed)` test also be removed, or is something else using the value it sets?

**Review by AndrewHess — COMMENTED** [review]
Looks good overall, I'm just a little confused about the changes to the workflow tests

File: .github/workflows/lint-test.yaml
Comment (srisub98) [reply .github/workflows/lint-test.yaml:1]: so we want to keep the lint step. 

I just removed the kind cluster & mock install as those would need s3 access for the multinode deployment to work and the updated helm chart doesn't allow for a single instance to be both query and ingest (can be extended later on if we want it)

Comment (AndrewHess) [reply .github/workflows/lint-test.yaml:1]: Ok that makes sense. I agree the lint test is good, but is the list-changed test actually testing anything? It looks like the only test that did require it is now removed. Or am I missing something?

Comment (srisub98) [reply .github/workflows/lint-test.yaml:1]: oh yea - I have removed the `changed` var in recent commit

**Pull request merged**